packages/eds-core-react/CHANGELOG.md

```diff
@@ -5,14 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.19.0] - 2022-04-06
+
+### Added
+
+- `Menu` now supports keyhandling on anchor element for open & focus on `Enter`/`ArrowUp`/`ArrowDown` ([#1936](https://github.com/equinor/design-system/issues/1936))
+
+  - Users now need to store anchor element in `useState` for correct re-renders & listening events
```

Comment:
Perhaps add a second bullet point that 'focus' attribute has been deprecated

Reply:
Fixed. Added a `Removed` section as per keep a changelog standard